.claude/agents/phoenix.md

```
@@ -83,12 +83,12 @@ For each refactoring:
 
 **ALWAYS write plan to:**
 ```
-$CLAUDE_PROJECT_DIR/thoughts/shared/plans/refactor-[target]-plan.md
+$CLAUDE_CC_DIR/thoughts/shared/plans/refactor-[target]-plan.md
 ```
 
 **Also write summary to:**
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)